setup.py

```diff
@@ -48,6 +48,9 @@ def get_version():
     ],
     setup_requires=[] + (['pytest-runner'] if 'test' in sys.argv else []) + [],
 
+    extras_require={
+        'telegram': ['python-telegram-bot']
```

Comment:
Do we need to pin down the lowest version number?

Reply:
Yes, but more important was to pin higher version because maintainers are working on major release version 12 with lot of changes.

Reply:
Are they already guilty in backward incompatible releases? %)

Reply:
I saw a lot of warnings in console for beta version so decide exclude 12 version for now.